### ISSUE TITLE
Add 128 bit private key import

### ISSUE THREAD

**Issue by europound** [TronLink/tronlink-extension#331]
Hello!
Could somebody Add 128 bit private key import please?

**Comment by sk4king** [comment]
Yes, please. I have an old private key from TronWatch Wallet and i need to import this wallet into TronLink
